pkg/decode/decode.go

```diff
@@ -11,10 +11,18 @@ import (
 
 type unmarshalFunc func([]byte, interface{}) error
 
+// FromJSON reads an asyncapi document from an input in a JSON format
+// and stores it in the value. If operation fails function returns an error.
```

Comment:
The same here: I'm wondering if it is `a value` or `the value` - do you mean any value or a particular one?

Reply:
It will be stored in `v interface{}`.
I guess it's `a` from method signature POV since it can be anything that is `interface{}'ish` =)

Reply:
As agreed offline we will use `the`.